common/lib/capa/capa/tests/test_shuffle.py

```diff
@@ -194,7 +194,10 @@ def test_shuffle_fixed_both_ends(self):
         """)
         problem = new_loncapa_problem(xml_str, seed=0)
         the_html = problem.get_html()
-        self.assertRegexpMatches(the_html, r"<div>.*\[.*'Alpha'.*'Beta'.*'B'.*'A'.*'C'.*'D'.*'Psi'.*'Omega'.*\].*</div>")
+        self.assertRegexpMatches(
```

Comment:
I saw that there is now a discussion about this, and one can rebase from master to get the increased limit. However, I'd rather just fix a single error.